tox.ini

```diff
@@ -1,11 +1,11 @@
 [tox]
-envlist=py{38,39,310,311,312,py3},alpine,flake8,checkmanifest,isort,mypy,doc8
+envlist=py{38,39,310,311,312,313,py3},alpine,flake8,checkmanifest,isort,mypy,doc8
 skip_missing_interpreters = true
 
 [testenv]
 deps =
     coverage>=5.2
-    freezegun<0.4
+    freezegun<2
```

Comment:
How is this related to the Python 3.13 change?

Reply:
Ah, forgot to comment. freezegun 1.5.0 was made compatible with Python 3.13 via https://github.com/spulec/freezegun/commit/d75458f3f691d0c2c35559872716d0ebab738817.

Reply:
Thoughts @stefan6419846?

Reply:
I am usually against needless lower pinning.

Reply:
Is there a specific reason for the upper limit? We tend to not have it for the other dependencies as well.

Reply:
Not really. I'd actually prefer `freezegun>=1.5.0` since that's really the hard constraint to make it work on 3.13.

Reply:
I've removed the upper constraint: 6c915fe0fc1505b65a247bb4c106cf9dbaf332f5